dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship/Functions/GetApprenticeshipsAsProvider.cs

```diff
@@ -37,23 +37,21 @@ public GetApprenticeshipsAsProvider(IAppCache cache, IApprenticeshipService appr
         [FunctionName("GetApprenticeshipsAsProvider")]
         public async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "bulk/providers")]HttpRequest req, ILogger log)
         {
-            List<Apprenticeship> persisted = null;
-
             try
             {
-                log.LogInformation($"[{DateTime.UtcNow:G}] Retrieving Apprenticeships...");
-                
-                persisted = (List<Apprenticeship>)await _apprenticeshipService.GetLiveApprenticeships();
-                
-                if (persisted == null)
-                {
-                    return new EmptyResult();
-                }
-
                 var result = await _cache.GetOrAddAsync<(string ExportKey, string Value)>("DasProviders", async () =>
                 {
+                    log.LogInformation($"Retrieving Apprenticeships...");
+
+                    var apprenticeships = (List<Apprenticeship>)await _apprenticeshipService.GetLiveApprenticeships();
```

Comment:
Why not change the interface to List instead of casting?
Using the cast is more fragile to future refactoring.

Reply:
I've not touched that line of code.

Reply:
ah okay, diff noise, :+1: